dev/resources/schema.setup.cql

```diff
@@ -5,11 +5,10 @@ CREATE TABLE IF NOT EXISTS lcmap_landsat_dev.sources (
     id text,
```

Comment:
do we add records for each progress update?  and if so, persist those indefinitely?